build.py

```diff
@@ -805,17 +812,34 @@ def TEMPFILE(cmd_, comstr=None):
                 ))
             return
         elif tool not in ('$CC', '$CXX', '$SHCC', '$SHCXX', '$LINK', '$SHLINK', '$AR', '$RC'):
+            list_targets = strmap(targets)
+            list_sources = strmap(sources)
+
             n.scan() # We need this for IDL.
             implicit_deps += strmap(n.implicit)
             self.builds.append(dict(
                 rule='EXEC',
-                outputs=strmap(targets),
-                inputs=strmap(sources),
+                outputs=list_targets,
+                inputs=list_sources,
                 implicit=implicit_deps,
                 variables={
                     'command': self.make_command(myEnv.subst(str(n.executor), executor=n.executor)),
                     }
                 ))
+
+            # Starting in SERVER-4304, there is no more install rules for unittests but there is a
```

Comment:
This ticket says is something about Visual Studio. Is this the right server ticket?

Reply:
I was missing a digit. Good catch